src/sonic-config-engine/sonic-cfggen

```diff
@@ -55,9 +56,9 @@ def main():
     group = parser.add_mutually_exclusive_group()
     group.add_argument("-t", "--template", help="render the data with the template file")
     group.add_argument("-s", "--alias-mapping", help="print alias mapping json if available", action='store_true')
-    group.add_argument("-v", "--var", help="print the value of a variable")
+    group.add_argument("-v", "--var", help="print the value of a variable, support jinja2 expression")
     group.add_argument("--var-json", help="print the value of a variable, in json format")
-    group.add_argument("--var-keys", help="print all keys of a map variable")
+    group.add_argument("--var-keys", help="print all keys of a map variable - to be deprecated, use -v and keys()")
```

Comment:
No other script is using, just to keep it there for a while in case someone is using it offline. To be removed in later version.